### DISCUSSION TITLE
Searchbar

### DISCUSSION THREAD

**Discussion by RSelwa** [dailydotdev/daily#993] [Feature Requests]
The searchbar could search articles related to research instead of chat gpt response, we google things that we nned to look but it would be cool to search through articles. For exemple database relational, I'm pretty sure that there some good articles about it but i don't care about the definition of wikipedia. Thanks

**Comment by nimrodkra** [comment]
Makes sense and many others have given similar feedback. No worries, our team is on it and we will soon add this functionality to the search bar as it’s pretty basic apparently. In the meantime, feel free to use the “Post finder” feature: https://app.daily.dev/posts/finder

Also I want to invite you to join the daily.dev ( http://daily.dev/ ) search Squad where we talk about the feature and share updates: https://dly.to/NJnTACdIfWV

**Reply by RSelwa** [reply]
Thanks for your response, I will use the post finder now 😉
